packages/design-system-react/src/components/button-primary/ButtonPrimary.stories.tsx

```diff
@@ -0,0 +1,180 @@
+      control: 'text',
+      description:
+        'Optional prop for additional CSS classes to be applied to the ButtonPrimary component',
+    },
+    isDanger: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, applies error/danger styling to the button',
+    },
+    isInverse: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, applies inverse styling to the button',
+    },
+    size: {
+      control: 'select',
+      options: Object.values(ButtonPrimarySize),
+      description: 'Optional prop to control the size of the ButtonPrimary',
+    },
+    isFullWidth: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, makes the button take up the full width of its container',
+    },
+    isLoading: {
+      control: 'boolean',
+      description: 'Optional prop that when true, shows a loading spinner',
+    },
+    loadingText: {
+      control: 'text',
+      description:
+        'Optional prop for text to display when button is in loading state',
+    },
+    startIconName: {
+      control: 'select',
+      options: Object.values(IconName),
+      description:
+        'Optional prop to specify an icon to show at the start of the button',
+    },
+    startIconProps: {
+      control: 'object',
+      description:
+        'Optional prop to pass additional properties to the start icon',
+    },
+    startAccessory: {
+      control: 'text',
+      description:
+        'Optional prop for a custom element to show at the start of the button',
+    },
+    endIconName: {
+      control: 'select',
+      options: Object.values(IconName),
+      description:
+        'Optional prop to specify an icon to show at the end of the button',
+    },
+    endIconProps: {
+      control: 'object',
+      description:
+        'Optional prop to pass additional properties to the end icon',
+    },
+    endAccessory: {
+      control: 'text',
+      description:
+        'Optional prop for a custom element to show at the end of the button',
+    },
+    isDisabled: {
+      control: 'boolean',
+      description: 'Optional prop that when true, disables the button',
+    },
+    loadingIconProps: {
+      control: 'object',
+      description:
+        'Optional prop to pass additional properties to the loading icon',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ButtonPrimary>;
+
+export const Default: Story = {
+  args: {
+    children: 'Primary Button',
+  },
+};
+
+export const IsDanger: Story = {
+  args: {
+    children: 'Danger Button',
+    isDanger: true,
+  },
+};
+
+export const IsInverse: Story = {
+  render: (args) => (
+    <div className="bg-primary-default p-4 rounded flex gap-2">
+      <ButtonPrimary {...args} isInverse>
+        Inverse Button
+      </ButtonPrimary>
+      <ButtonPrimary {...args} isInverse isDanger>
+        Inverse Danger Button
+      </ButtonPrimary>
+    </div>
+  ),
+};
```

Comment:
These stories cover the additional `isInverse` and `isDanger` props. The rest remain the same as `ButtonBase`, with the component and enums updated to `ButtonPrimary`.